ansible/roles/ansible_service_broker_setup/tasks/main.yml

```diff
@@ -59,6 +59,7 @@
         -p LAUNCH_APB_ON_BIND={{ broker_launch_apb_on_bind }}
         -p OUTPUT_REQUEST={{ broker_output_request }}
         -p RECOVERY={{ broker_recovery }}
+        -p IMAGE_PULL_POLICY={{ broker_pull_policy }}
```

Comment:
Please move this to the below section for the non-rcm build options.

Reasoning is we run the older 'release-0.9' branch of the deployment template for the --rcm option:
  https://github.com/openshift/ansible-service-broker/blob/release-0.9/templates/deploy-ansible-service-broker.template.yaml

So we need to guard against newer options for the template and put them in the second block forming up the process command to run on the template.